### ISSUE TITLE
Notification Popup Persists (and prevents Blank Screen and Suspend)

### ISSUE THREAD

**Issue by Mikuana** [pop-os/pop#2516]
**Distribution (run `cat /etc/os-release`):**
 
 NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

geary:
  Installed: 40.0-2
  Candidate: 40.0-2
  Version table:
 *** 40.0-2 500
        500 http://us.archive.ubuntu.com/ubuntu jammy/universe amd64 Packages
        100 /var/lib/dpkg/status

**Issue/Bug Description:**

If I am not interacting with my computer (moving the mouse around, typing, etc.) the popup notification pops up, then never goes away. This will wake up the screen if it is blank, and prevent the screen from going blank again. This also prevents the system from automatically suspending. I have seen this happen the most with notifications from geary (since emails tend the generate the most notifications on my system), but I suspect this is an issue with any popup and is not specific to geary.

In other words, if I get an email after I walk away from my computer, it will leave the screen on forever.

**Steps to reproduce (if you know):**

Enable all notifications for Geary, except "Show Message Content on Lock Screen". Set everything else to on. Send email to yourself (from your phone, so that you're not using your PC). See the popup, don't touch anything. The popup will remain and the system will remain "active", so the blank screen and suspend behavior are never activated.

**Expected behavior:**

The popup will clear itself after a time, regardless of whether you're interacting with the system, and blank screen and suspend will work normally.
